hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OzoneFSInputStream.java

```diff
@@ -76,4 +79,21 @@ public boolean seekToNewSource(long targetPos) throws IOException {
   public int available() throws IOException {
     return inputStream.available();
   }
+
+  /**
+   * @param buf the ByteBuffer to receive the results of the read operation.
+   * @return the number of bytes read, possibly zero.
```

Comment:
According to the ByteBufferReadable javadoc, it should return -1 if end of file.

Reply:
Thanks @jojochuang for the review.
Fixed the description, and updated the implementation.